.github/workflows/fcos-podman-next-build.yml

```diff
@@ -17,36 +19,66 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - name: Install qemu dependency
+      run: |
+        sudo apt update
+        sudo apt -y install qemu-user-static
```

Comment:
IMHO, these two commands (esp. the first one) will introduce a non-trivial amount of  unpredictability in this workflow compared to `fcos-podman-next-build-prepush-test.yml`.  Would you consider doing this step, and the multi-arch build there as well?

The thing I'm afraid of, is the ability to notice a broken workflow early - within a PR,  rather than after merge.  From that perspective it would be good to keep the two workflows as identical as possible.

I'd even suggest (maybe for a future update), usinga local "reusable workflow" to de-duplicate at least the build portion.  There's [a simple example of doing this in the `podman.io` repo.](https://github.com/containers/podman.io/tree/636ba42ba742d05fdfb7090631d42e0f8d4bdf79/.github/workflows) - both `deploy.yml` and `test.yml` reuse `setup_and_build.yml`.

Reply:
multi-arch build takes an awful long and it might end up being an annoyance if we do it before merge. I haven't yet checked if it takes longer than the rest of podman tests combined.

Reply:
Yes, the GHA VMs are slow *AND* the emulation is slow as well.  This was one of the reasons I did this in Cirrus, since we can simply "throw more hardware" at the problem.  I'm not sure that's a possible solution here w/o TONs of more work securing it all.

Reply:
ack, let's leave any possible reusability for another followup later on. I can add comments in the actions file.